### PR TITLE
Properly tier National Dex

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: config/formats.ts [hunk 1]
```diff
@@ -530,17 +520,7 @@ export const Formats: FormatList = [
...
-			'Blissey', 'Chansey', 'Charizard-Mega-X', 'Charizard-Mega-Y', 'Clefable', 'Corviknight', 'Diancie-Mega', 'Excadrill', 'Ferrothorn',
-			'Garchomp', 'Gliscor', 'Greninja', 'Heatran', 'Kartana', 'Kommo-o', 'Kyurem', 'Landorus-Therian', 'Lopunny-Mega', 'Magnezone',
-			'Mawile-Mega', 'Pelipper', 'Rillaboom', 'Scizor-Mega', 'Serperior', 'Slowbro-Base', 'Swampert-Mega', 'Tapu Fini', 'Tapu Koko',
-			'Tapu Lele', 'Toxapex', 'Tyranitar', 'Victini', 'Volcarona', 'Weavile', 'Zapdos-Base',
-			'ND UUBL', // National Dex UUBL
-			'Drizzle', 'Drought', 'Light Clay',
-			// Slowbronite is banned so it doesn't validate on Galarian Slowbro
-			'Slowbronite',
-		],
+		banlist: ['ND UUBL', 'Drizzle', 'Drought', 'Light Clay', 'Slowbronite'],
```
Comment (urkerab) [inline config/formats.ts:523]:
```suggestion
		banlist: ['ND OU', 'ND UUBL', 'Drizzle', 'Drought', 'Light Clay', 'Slowbronite'],
```

File: config/formats.ts [hunk 2]
```diff
@@ -551,14 +531,7 @@ export const Formats: FormatList = [
...
 		ruleset: ['[Gen 8] National Dex UU'],
-		banlist: [
-			// National Dex UU
-			'Aegislash', 'Altaria-Mega', 'Amoonguss', 'Beedrill-Mega', 'Breloom', 'Celesteela', 'Chandelure', 'Donphan', 'Dragonite',
-			'Hatterene', 'Hippowdon', 'Infernape', 'Krookodile', 'Mamoswine', 'Mandibuzz', 'Manectric-Mega', 'Mienshao', 'Moltres-Base',
-			'Nidoking', 'Nihilego', 'Pidgeot-Mega', 'Regieleki', 'Rotom-Heat', 'Rotom-Wash', 'Salamence', 'Scizor', 'Skarmory', 'Slowking',
-			'Swampert', 'Urshifu-Rapid-Strike', 'Zeraora',
-			'ND RUBL', // National Dex RUBL
-		],
+		banlist: ['ND RUBL'],
```
Comment (urkerab) [inline config/formats.ts:534]:
```suggestion
		banlist: ['ND UU', 'ND RUBL'],
```

File: config/formats.ts (hunk 1 above)
Comment (KrisXV) [reply config/formats.ts:523]: lol oops th

Comment (KrisXV) [reply config/formats.ts:523]: Anks

**Pull request merged**